app/forms/apo_form.rb

```diff
@@ -65,6 +65,12 @@ def default_workflow
     model.administrativeMetadata.ng_xml.xpath('//registration/workflow/@id').to_s
   end
 
+  def agreement
+    return if new_record?
```

Comment:
why do we need to check if it's a new record or not? Can we just `delegate :agreement, to: :model`?

Reply:
makes sense

Reply:
after second look, delegation won't work currently since we don't have an associated model on the new action